src/utils/trust/mapper.to.session.ts

```diff
@@ -0,0 +1,19 @@
+import * as Trust from 'model/trust.model';
+import * as Page from 'model/trust.page.model';
+
+const mapDetail = (
```

Comment:
naming - can you rename mapDetail to be 'mapToTrust' as it will accept TrustDetails as param and return Trust